src/provision/freedom-module.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "provision",
+  "description": "Provisions VMs on Digital Ocean",
```

Comment:
This is fine but FYI I've generally found it easier to create a freedomjs module *after* the fact, i.e. once we know we need it. As I said in another comment, there's quite a lot of dependencies here so we might end up drawing the module boundary at just the social provider layer, instead, for module size if nothing else. Anyway, not a big deal for now.